src/server/scripts/Spells/spell_quest.cpp

```diff
@@ -1958,128 +1958,239 @@ class spell_q13086_cannons_target : public SpellScriptLoader
+    SPELL_EXPLODE_TROLL_MEAT              = 52578,
+    SPELL_EXPLODE_TROLL_BLOODY_MEAT       = 52580,
+
+    SPELL_BURST_AT_THE_SEAMS_59576        = 59576, //script/knockback, That's Abominable
+    SPELL_BURST_AT_THE_SEAMS_59579        = 59579, //dummy
+    SPELL_BURST_AT_THE_SEAMS_52510        = 52510, //script/knockback, Fuel for the Fire
+    SPELL_BURST_AT_THE_SEAMS_52508        = 52508, //damage 20000
+    SPELL_BURST_AT_THE_SEAMS_59580        = 59580, //damage 50000
+
+    SPELL_ASSIGN_GHOUL_KILL_CREDIT_TO_MASTER = 59590,
+    SPELL_ASSIGN_GEIST_KILL_CREDIT_TO_MASTER = 60041,
+    SPELL_ASSIGN_SKELETON_KILL_CREDIT_TO_MASTER = 60039,
+
+    SPELL_DRAKKARI_SKULLCRUSHER_CREDIT    = 52590,
+    SPELL_SUMMON_DRAKKARI_CHIEFTAIN       = 52616,
+    SPELL_DRAKKARI_CHIEFTAINK_KILL_CREDIT = 52620
 };
 
-class spell_q13264_thats_abominable : public SpellScriptLoader
+class spell_q13264_q13276_q13288_q13289_burst_at_the_seams_59576 : public SpellScriptLoader
 {
     public:
-        spell_q13264_thats_abominable() : SpellScriptLoader("spell_q13264_thats_abominable") { }
+        spell_q13264_q13276_q13288_q13289_burst_at_the_seams_59576() : SpellScriptLoader("spell_q13264_q13276_q13288_q13289_burst_at_the_seams_59576") { }
 
-        class spell_q13264_thats_abominable_SpellScript : public SpellScript
+        class spell_q13264_q13276_q13288_q13289_burst_at_the_seams_59576_SpellScript : public SpellScript
         {
-            PrepareSpellScript(spell_q13264_thats_abominable_SpellScript);
+            PrepareSpellScript(spell_q13264_q13276_q13288_q13289_burst_at_the_seams_59576_SpellScript);
 
             bool Validate(SpellInfo const* /*spellInfo*/) override
             {
                 return ValidateSpellInfo(
                 {
-                    SPELL_ICY_GHOUL_CREDIT,
-                    SPELL_VICIOUS_GEISTS_CREDIT,
-                    SPELL_RISEN_ALLIANCE_SOLDIERS_CREDIT,
+                    SPELL_BURST_AT_THE_SEAMS_59576,
+                    SPELL_BLOATED_ABOMINATION_FEIGN_DEATH,
+                    SPELL_BURST_AT_THE_SEAMS_59579,
+                    SPELL_BURST_AT_THE_SEAMS_BONE,
+                    SPELL_EXPLODE_ABOMINATION_MEAT,
+                    SPELL_EXPLODE_ABOMINATION_BLOODY_MEAT
                 });
             }
 
-            void HandleKnockBack(SpellEffIndex effIndex)
+            void HandleScript(SpellEffIndex /*effIndex*/)
             {
-                PreventHitDefaultEffect(effIndex);
-
-                if (Creature* creature = GetHitCreature())
-                    if (Unit* charmer = GetCaster()->GetCharmerOrOwner())
-                        if (Player* player = charmer->ToPlayer())
-                            if (player->GetQuestStatus(QUEST_THATS_ABOMINABLE) == QUEST_STATUS_INCOMPLETE)
-                                if (GiveCreditIfValid(player, creature))
-                                    creature->KillSelf();
+                if (Creature* creature = GetCaster()->ToCreature()) {
```

Comment:
{  should be below

Reply:
yup, TC codestyle dictates that brackets should be on separate lines.